In [11]:
# Importing libraries
import pandas as pd
from sklearn.model_selection import train_test_split
import numpy as np

In [6]:
# Importing dataset
df = pd.read_csv('Iris.csv', index_col=0)

In [7]:
# Looking for columns and rows
df.shape

(150, 5)

This shows that this dataset has 150 records(rows) and 6 features(columns).

In [8]:
# I/O data
X = df.iloc[:,:-1]
y = df.iloc[:,-1]

In [26]:
# Creating dataframe to save data.
column_names = ['X_train', 'y_train', 'X_test', 'y_test']
train_test = pd.DataFrame(columns=column_names)

In [35]:
# Using for loop creating 4 different subsets.
for i in np.arange(50,70,5):
    train_test.append(train_test_split(X, y, test_size=0.18, random_state=i))

In [36]:
print(train_test)

[     SepalLengthCm  SepalWidthCm  PetalLengthCm  PetalWidthCm
Id                                                           
142            6.9           3.1            5.1           2.3
60             5.2           2.7            3.9           1.4
40             5.1           3.4            1.5           0.2
94             5.0           2.3            3.3           1.0
145            6.7           3.3            5.7           2.5
..             ...           ...            ...           ...
72             6.1           2.8            4.0           1.3
41             5.0           3.5            1.3           0.3
64             6.1           2.9            4.7           1.4
118            7.7           3.8            6.7           2.2
47             5.1           3.8            1.6           0.2

[123 rows x 4 columns],      SepalLengthCm  SepalWidthCm  PetalLengthCm  PetalWidthCm
Id                                                           
128            6.1           3.0            